app/services/data/get-staging-court-reports.js

```diff
@@ -0,0 +1,74 @@
+const knex = require('../../../knex').stagingSchema
+const Promise = require('bluebird').Promise
+const OmCourtReports = require('wmt-probation-rules').OmCourtReports
+const CasesSummary = require('wmt-probation-rules').CasesSummary
+const CourtReport = require('wmt-probation-rules').CourtReport
+
+module.exports = function (range) {
+  var omCourtReports = []
+
+  var tableName = 'court_reporters'
+  var selectCols = [
+    'id AS staging_id',
+    'trust',
+    'region_desc',
+    'region_code',
+    'ldu_desc',
+    'ldu_code',
+    'team_desc',
+    'team_code',
+    'om_surname',
+    'om_forename',
+    'om_grade_code',
+    'om_key',
+    'sdr_last_30',
+    'sdr_due_next_30',
+    'sdr_conv_last_30',
+    'oral_reports',
+    'datestamp'
+  ]
+
+  return knex(tableName).whereBetween('id', range).select(selectCols)
+  .then(function (results) {
+    if (results !== 'undefined' && results.length > 0) {
+      return Promise.each(results, function (result) {
+        var casesSummary = new CasesSummary(
+          result['trust'],
+          result['region_desc'],
+          result['region_code'],
+          result['ldu_desc'],
+          result['ldu_code'],
+          result['team_desc'],
+          result['team_code'],
+          result['om_surname'],
+          result['om_forename'],
+          result['om_grade_code'],
+          result['om_key'],
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null
```

Comment:
Don't think the `null`s are required

Reply:
If we don't pass in null then the fields will just be undefined.  I think I prefer null to undefined?

Reply:
👍